evennia/accounts/accounts.py

```diff
@@ -323,6 +329,7 @@ def unpuppet_object(self, session):
                 obj.sessions.remove(session)
                 if not obj.sessions.count():
                     del obj.account
+                OBJECT_UNPUPPET.send(sender=obj, session=session, account=self)
```

Comment:
Moving this below the hook and calling it `OBJECT_POST_UNPUPPET` seems clearer. Or maybe naming these signals the same as the hooks (`AT_POST_UNPUPPET`)?